Resources/doc/reference/installation.rst

```diff
@@ -188,10 +188,14 @@ Add the related security routing information:
 
     # app/config/routing.yml
 
-    sonata_user:
+    sonata_user_admin_security:
```

Comment:
i would not change this, this is a BC break IMO

@Soullivaneuh @rande what do you mean?

otherwise i agree to make this BC break, but maybe in a separate PR?